cmd/check/check.go

```diff
@@ -41,6 +42,7 @@ func init() {
 // AddFlags adds the check flags to the cmdFlags command
 func AddFlags(cmdFlags *pflag.FlagSet) {
 	flags.BoolVarP(cmdFlags, &oneway, "one-way", "", oneway, "Check one way only, source files must exist on remote", "")
+	flags.BoolVarP(cmdFlags, &ReportDirs, "report-dirs", "", ReportDirs, "Report directory differences in addition to files", "")
```

Comment:
Naming things... We seem to spell out words like "directory" for most flags, and even for crypt backend we have this rather long `--crypt-directory-name-encryption`. However, its not very consistent, as for config we have two flags `--temp-dir` and `--cache-dir`. So I'm thinking if this should be `--report-directories`, or could it simply be `--directories`?

Reply:
Either of those would be fine with me! My thought was that `dirs` is consistent with `--create-empty-src-dirs` (the closest comparable flag) and also that shorter is better when it comes to flag names. But I am happy to change it!

Reply:
Ah, right, that is one command flag using the short, and plural, "dirs". I looked primarily at global and backend flags, I think. So it is even more mixed than I initially thought.

Reply:
He, he!

`--report-dirs` I think is OK! At some point I want to put these flags on sync/copy/move and for that `--dirs` is too generic.